catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/types/ValidationAttributes.java

```diff
@@ -49,6 +49,22 @@
             true /* tokenized */,
             true /* multivalued */,
             BasicTypes.STRING_TYPE));
+    descriptors.add(
+        new AttributeDescriptorImpl(
+            Validation.FAILED_VALIDATORS_WARNINGS,
+            true /* indexed */,
+            true /* stored */,
+            false /* tokenized */,
```

Comment:
If these are not tokenized doesn't that mean that knowledge managers will not be able to match keywords in these fields? Constructing searches against exact error messages is pretty tedious.

Reply:
Up until now there has been a version of the descriptor that is tokenized and a version that is not. This was my concern: how do we know which one is correct if our system has been using both variations for the past couple of releases?